### ISSUE TITLE
Stakeholder response from `Flowee`

### ISSUE THREAD

**Issue by tomFlowee** [bitjson/bch-vm-limits#37]
https://codeberg.org/bitcoincash/CHIP-VM-Limits/commits/branch/flowee

**Comment by bitjson** [comment]
Thanks for opening an issue @tomFlowee! Looking at the latest commit there:

<img width="1181" alt="Screenshot 2024-10-30 at 1 27 09 PM" src="https://github.com/user-attachments/assets/a5add716-6831-411e-ad2f-8ea720efaf32">

It looks like you marked the `Approve` and `Disapprove` columns (but not the `Neutral` column) with the icon for neutral responses (`➖`), and your statement is also placed above the section for `Approve` statements. 

Just to clarify, this is a `Neutral` response for `Flowee`, `Flowee Pay`, and `Flowee Products`?

And can you confirm this statement is `Neutral`?

> The ideas look very good and clearly intend to make the capabilities next gen, which is great. The proposal is lacking in the parts where we can understand the actual impact to scaling and operations.
>
> There isn't any wall-clock analysis to that should be front and center on the risk-assessment side. Without putting in the hours ourselves it is nigh impossible to figure out how long it will take to validate transactions or blocks that push the boundaries of what is allowed.
>
> We can't approve it without such basic analysis, but giving a vote of disapproval is also too far, the benefits are great. Maybe we'll see a report on actual impact (linking to data) before the May upgrade. That would be ideal.
>
> —<cite>[Flowee](https://flowee.org)</cite>

Thanks!

**Comment by tomFlowee** [comment]
The comments try to make clear that there are negative issues and there are positives. It is really not possible to at this time file it under any of the 3 columns exclusively. Neutral doesn't make sense since there IS a comment added which is not neutral.

So the various things you commented on are intended to reflect the fact that it is none of those.

The comment itself is filed under the header "the following statements..." for the same reason.

Does that help?

**Comment by bitjson** [comment]
A public statement may be provided regardless of whether Flowee's response is `Yes (Approve)`, `No (Disapprove)`, or `Abstain (Neutral)`.

The formal request again (from https://codeberg.org/Flowee/thehub/issues/6): 

<blockquote>

Does Flowee approve of activating "CHIP-2021-05 VM Limits: Targeted Virtual Machine Limits" and "CHIP-2024-07 BigInt: High-Precision Arithmetic for Bitcoin Cash" in the May 2025 Upgrade to Bitcoin Cash (BCH)?

- Yes (Approve)
- No (Disapprove), or
- Abstain (Neutral)

Separate responses may also be given for each CHIP. To make a public statement explaining this decision, please provide it here (required for disapprovals): <br/><br/>

</blockquote>

Reading your statement again, I see, `We can't approve it without such basic analysis [...]`. 

Can I mark `Flowee`, `Flowee Pay`, and `Flowee Products` as `No (Disapprove)`, and file your statement under the `Disapprove` heading?

**Comment by tomFlowee** [comment]
I understand the complexity of the process doesn't fit in neat boxes. Please don't try to make them fit and break the intent here.

This statement is given in text in order to make clear what is going on. It is neither approve nor disapprove..  Let me re-iterate this to make it clear.

Your 2nd time asking if this fits in your neat columns is very strict and very beaurocratic, sorry that isn't always going to be how the real world works.

What is wrong with the 2 dashes I wrote? Why are you insisting it change?

**Comment by tomFlowee** [comment]
Rebased and forced-pushed the branch.

I haven't received an answer to my question of what is wrong, so I'm assuming it is correct.
Please pull the response.

**Comment by bitjson** [comment]
@tomFlowee responding here one more time following your [Telegram message](https://t.me/bchbuilders/33855). 

I will not restructure this document to incorporate ambiguous responses. 

I've donated many one-on-one hours to your understanding and review of this proposal over multiple months – I will not be donating further time.

Given your repeated public misdirection and antagonism, it seems like your response is `No (Disapprove)`. 

**Please confirm here so I can update the CHIP.**

> The CHIP contributors believe Flowee is an important stakeholder in the Bitcoin Cash ecosystem. **Please respond by 12:00 UTC on November 14, 2024; non-responses will be considered "Abstain (Neutral)".**

If you've still failed to provide a non-ambiguous response by the deadline, `Flowee`, `Flowee Pay`, and `Flowee Products` will be marked `Abstain (Neutral)` as explained in the formal request.

**Comment by tomFlowee** [comment]
Hey Jason.

It is just a merge, man. No need to "restructure" anything. Nobody is suggesting that...

This statement is given in text in order to make clear what is going on. It is neither approve nor disapprove.. Let me re-iterate this to make it clear.

**Comment by bitjson** [comment]
Given your continued misdirection here and [on Telegram](https://t.me/bchbuilders/33862), **I'm going to make the call that your response is `No (Disapprove)` if you haven't provided a different, clear response before the deadline (Nov 14)**. 

Normally, I'd mark non-responsive stakeholders as `Abstain (Neutral)`, but in this particular case, I'm no longer willing to risk this being misinterpreted as an attempt to hide a non-approving response.

So, again: 

- I will not be reformatting or adding any new sections to `stakeholders.md` on account of this issue.
- I do not plan to comment further in this issue other than to answer good-faith questions that haven't already been answered. (**And I may hide further responses to ensure this comment remains the most recent.**)
- **If you have not provided an unambiguous `Yes (Approve)` or `Abstain (Neutral)` response by the deadline, I will consider `Flowee`, `Flowee Pay`, and `Flowee Products` to have responded with `No (Disapprove)`, and I will include your provided statement under the `Disapprove` heading.**

**Comment by tomFlowee** [comment]
When I asked you a week ago "What is wrong with the 2 dashes I wrote? Why are you insisting it change?", you never answered.  Me following up now leads to rather agressive posts from you.  
I don't understand why.  
The issues I addressed are real issues, I would like to not focus on the dash needing to be a checkbox in your document. That is soo not relevant in the bigger scope! Why get upset over that?

The bottom line is: I don't think the proposal warrents a rejection, I don't think it is a clear "ok" either.
And this is not the first time I addressed this. So it should not be a surprise that I wrote that.

I'd propose a more productive use of time:
What I'd like to ask is if you can address the issues and work on those. Make the problem go away.

**Comment by bitjson** [comment]
Marked as `Disapprove` in https://github.com/bitjson/bch-vm-limits/commit/9994a9b6f11b2e8e7fa24aac868e4809e6338989, but if that changes before the deadline, please let me know @tomFlowee
